bin/growpart

```diff
@@ -744,6 +744,14 @@ has_cmd() {
 	command -v "${1}" >/dev/null 2>&1
 }
 
+cmd_is_busybox() {
+	if [ "$(basename $(readlink -f $(command -v "${1}")))" = "busybox" ]; then
+		return 0
```

Comment:
this doesn't check for error of any of those commands.  it'll work, but its wierd.  if any of those commands fail you'll end up with a strange thing that does [ "" = "busybox" ]. 

I personally prefer this:

```bash
local out="" target=""
out=$(command -v "$1") && target=$(readlink -f "$out") &&
    [ "${target##*/}" = "busybox" ] && return 0
```

But I wont insist on it.  I only really commented the first time because I was pretty sure it didn't work.
what you have above will work, but I really try to stay away from things like that.

Reply:
OK, happy to change it to that.